.travis.yml

```diff
@@ -122,13 +120,39 @@ install:
 script:
   - ./bootstrap
   - ./configure $FFCONFIG
-  - make
-  - make install
+  - make -j$(nproc)
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then mkdir -p appdir ; make install DESTDIR=$(readlink -f appdir); fi
+  - if [ "$TRAVIS_OS_NAME" != "linux" ]; then make install; fi
   - make check
   - if [ ! -z "$LINUX_FULL" ]; then make distcheck; fi
   - fontforge -version
   - $PYTHON -c "import fontforge; import psMat; print(fontforge.__version__, fontforge.version()); fontforge.font();"
   - if [ "$TRAVIS_OS_NAME" == "osx" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then travis-scripts/ffosxbuild.sh $PREFIX $TRAVIS_COMMIT; fi
+  - |
+    if [ ! -z "$LINUX_FULL" ] ; then
+    mkdir -p ./appdir/usr/share/applications/ # FIXME: #3372
+    cp desktop/fontforge.desktop ./appdir/usr/share/applications/ # FIXME: #3372
+    mkdir -p ./appdir/usr/share/icons/hicolor/ # FIXME: #3372
+    cp -r desktop/icons/* ./appdir/usr/share/icons/hicolor/ ; rm -rf ./appdir/usr/share/icons/hicolor/src # FIXME: #3372
+    mkdir -p appdir/usr/share/fontforge/hotkeys ; cp share/default appdir/usr/share/fontforge/hotkeys/ # FIXME: #3372
+    find .  | grep "pixmaps/resources"
+    mkdir -p appdir/usr/share/fontforge/pixmaps ; find . -type f -name "resources" -exec cp {} appdir/usr/share/fontforge/pixmaps/ \;
+    # TODO: AppStream metainfo
+    ( cd appdir ; dpkg -x /var/cache/apt/archives/libpython3.6-minimal*.deb . )
+    ( cd appdir ; dpkg -x /var/cache/apt/archives/libpython3.6-stdlib*.deb . )
+    find appdir/
+    wget -c -nv "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
+    chmod a+x linuxdeployqt-continuous-x86_64.AppImage
+    export VERSION=$(git rev-parse --short HEAD) # linuxdeployqt uses this for naming the file
+    ./linuxdeployqt*.AppImage ./appdir/usr/share/applications/*.desktop -bundle-non-qt-libs
+    # Manually invoke appimagetool so that the custom AppRun stays intact
+    ./linuxdeployqt*.AppImage --appimage-extract
+    export PATH=$(readlink -f ./squashfs-root/usr/bin):$PATH
+    rm appdir/AppRun ; cp Packaging/AppDir/AppRun appdir/AppRun ; chmod +x appdir/AppRun # custom AppRun
+    ./squashfs-root/usr/bin/appimagetool -g ./appdir/
+    find appdir -executable -type f -exec ldd {} \; | grep " => /usr" | cut -d " " -f 2-3 | sort | uniq
+    curl --upload-file FontForge*.AppImage https://transfer.sh/FontForge-git.$(git rev-parse --short HEAD)-x86_64.AppImage # For testing only
```

Comment:
You can upload to bintray, we do this for the osx CI builds

Reply:
How do we tell `provider: bintray` which files to upload?